libraries/ui/src/main/java/androidx/media3/ui/SubtitlePainter.java

```diff
@@ -44,12 +47,16 @@
 import org.checkerframework.checker.nullness.qual.MonotonicNonNull;
 import org.checkerframework.checker.nullness.qual.RequiresNonNull;
 
-/** Paints subtitle {@link Cue}s. */
+/**
+ * Paints subtitle {@link Cue}s.
+ */
 /* package */ final class SubtitlePainter {
```

Comment:
Please do not format the media3 source file, as this will cause more conflicts when we upgrade media3 to latest, which will make the upgrade more difficult. When we upgrade media3 from 1.0.2 to 1.2.1 we merged about 2000+ files.

Reply:
resolved